actions/maven/publish/action.yml

```diff
@@ -0,0 +1,114 @@
+# Copyright 2023 SLSA Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
```

Comment:
Can you add a README.md for the Action itself?

Reply:
FWIW, we haven't done this for other actions. I agree we should have it (and for other actions) but I personally wouldn't consider it a blocker.

Reply:
true. Since the Gradle Action has a README, let's be consistent